src/olympia/amo/management/commands/check_locales_completion_rate.py

```diff
@@ -0,0 +1,175 @@
+            self.find_locales_below_and_above_threshold()
+        )
+        self.warn_about(
+            locales_below_threshold=locales_below_threshold,
+            locales_above_threshold=locales_above_threshold,
+        )
+
+    def find_locales_below_and_above_threshold(self):
+        locales_below_threshold = set()
+        locales_above_threshold = set()
+        seen_locales = defaultdict(int)
+        for project in self.PONTOON_PROJECTS:
+            data = self.fetch_data(project)
+            for locale_data in data.get('project', {}).get('localizations', []):
+                locale = self.get_locale(locale_data)
+                seen_locales[locale] += 1
+                completion = self.get_completion(locale_data)
+                # If we see a locale below treshold, immediately add it to the
+                # relevant set.
+                if completion < self.COMPLETION_THRESHOLD:
+                    self.stdout.write(f'❌ {locale} is below threshold ({completion}%)')
+                    locales_below_threshold.add(locale)
+                # If we see a language above threshold but not already enabled
+                # in production, add it to the relevant set.
+                elif locale not in settings.AMO_LANGUAGES:
+                    self.stdout.write(f'✅ {locale} is above threshold ({completion}%)')
+                    locales_above_threshold.add(locale)
+        # Add to locales below threshold locales we only saw in one project.
+        # Use the ones we already have enabled in production, in case somehow
+        # a locale would disappear completely from pontoon and still be enabled
+        # on our side.
+        for locale in settings.AMO_LANGUAGES:
+            if locale != settings.LANGUAGE_CODE and seen_locales[locale] != len(
+                self.PONTOON_PROJECTS
+            ):
+                self.stdout.write(
+                    f'❌ {locale} is only in {seen_locales[locale]} project(s) !'
+                )
+                locales_below_threshold.add(locale)
+        # Somewhat inefficient but good enough check to remove any locales that
+        # both below and above threshold because of differences between the
+        # projects.
+        for locale in locales_above_threshold:
+            if locale in locales_below_threshold:
+                self.stdout.write(f'❌ {locale} is in both sets !')
+                locales_above_threshold.remove(locale)
+        return locales_below_threshold, locales_above_threshold
+
+    def fetch_data(self, project):
+        q = re.sub(r'\s', '', self.QUERY_TEMPLATE % {'project': project})
+        self.stdout.write(f'Calling pontoon with {q}')
+        response = requests.get(self.PONTOON_API, {'query': q})
+        response.raise_for_status()
+        data = response.json().get('data', {})
+        return data
+
+    def get_completion(self, locale_data):
+        return round(locale_data['approvedStrings'] / locale_data['totalStrings'] * 100)
```

Comment:
if `totalStrings` is ever zero, this could raise. might want to have a quick check for it.

Reply:
And do what though ? If pontoon is reporting a `0` here I'm okay with raising: this should be reported in Sentry, it's unexpected behavior.

Reply:
If it is 0 you could just return 0. That would indicate there are no translations for a locale right? Since this is a job that runs automatically and regularly might be good to avoid typical cases where it could raise.

Reply:
I would prefer raising, because it's not expected behavior. If we return `0` then it will impact the output and if it's a global bug affecting multiple locales we'd never know which locales should legitimately be removed or which just had a bug